build.sbt

```diff
@@ -70,12 +70,14 @@ lazy val edu_gemini_seqexec_web = project.in(file("modules/edu.gemini.seqexec.we
   .settings(commonSettings: _*)
   .enablePlugins(AutomateHeaderPlugin)
   .enablePlugins(GitBranchPrompt)
+  .disablePlugins(RevolverPlugin)
```

Comment:
Thanks to sbt magic, the revolver plugin gets confused when adding scalafix 🤷‍♂️